Source/WebKit/Shared/CoordinatedGraphics/threadedcompositor/ThreadedCompositor.cpp

```diff
@@ -276,7 +277,25 @@ void ThreadedCompositor::renderLayerTree()
     m_scene->applyStateChanges(states);
     m_scene->paintToCurrentGLContext(viewportTransform, FloatRect { FloatPoint { }, viewportSize }, m_flipY);
 
-    m_context->swapBuffers();
+#if ENABLE(BUFFER_DAMAGE_TRACKING)
+    if (!m_scene->lastDamagedRects().isEmpty()) {
+        RELEASE_LOG(CoordinatedGraphics, "damaged regions:");
```

Comment:
Please add this change:

```diff
diff --git a/Source/WebKit/Shared/CoordinatedGraphics/threadedcompositor/ThreadedCompositor.cpp b/Source/WebKit/Shared/CoordinatedGraphics/threadedcompositor/ThreadedCompositor.cpp
index eabac2fe7ade..fa6e33edaa90 100644
--- a/Source/WebKit/Shared/CoordinatedGraphics/threadedcompositor/ThreadedCompositor.cpp
+++ b/Source/WebKit/Shared/CoordinatedGraphics/threadedcompositor/ThreadedCompositor.cpp
@@ -29,6 +29,8 @@
 #if USE(COORDINATED_GRAPHICS)
 
 #include "CompositingRunLoop.h"
+#include "Logging.h"
+
 #include <WebCore/NicosiaBufferDamage.h>
 #include <WebCore/PlatformDisplay.h>
 #include <WebCore/TransformationMatrix.h>
```

Otherwise you will got a build error with the `ENABLE(BUFFER_DAMAGE_TRACKING)` enabled.